File: JAVA/Program38.java
//////////////////////////////////////////////////////////////////////////////////////////
//                                                                                      //
//  File name :     Program38.java                                                      //
//  Description :   Used to Print Given Pattern                                         //
//  Author :        Pallavi Omprakash Malewar                                           //
//  Date :          25/08/2025                                                          //
//                                                                                      //
//////////////////////////////////////////////////////////////////////////////////////////



// Input : 4    4
// Output :
/*
    a   a   a   a
    b   b   b   b
    c   c   c   c
    d   d   d   d
*/
            

import java.util.Scanner;

class Pattern
{
    public void Display(int iRow , int iCol)
    {
        int i = 0, j = 0;
        char ch = 'a';

        for(i = 1, ch = 'a'; i <= iRow ; i++, ch++)
        {
            for(j = 1 ; j <= iCol ; j++)
            {
                System.out.print(ch + "\t");
            }
            System.out.println();
        }
    }
}

class Program38
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.print("Enter Number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter Number of Columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}

File: CPP/Program38.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////
//                                                                                                 //
//  File name :     Program38.cpp                                                                  //
//  Description :   Used to dynamically allocate the memory in an integer Array                    //
//  Author :        Pallavi Omprakash Malewar                                                      //
//  Date :          06/09/2025                                                                     //
//                                                                                                 //
/////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

int main()
{
    int *ptr = NULL;
    int iLength = 0 , iCnt = 0;

    cout<<"Enter Number of Elements : ";
    cin>>iLength;

    ptr = new int[iLength];     //Dynamic Memory Allocation

    cout<<"Enter the Elements :\n";
    for(iCnt = 0 ; iCnt < iLength ; iCnt++)
    {
        cin>>ptr[iCnt];
    }

    cout<<"Elements of the Array :\n";
    for(iCnt = 0 ; iCnt < iLength ; iCnt++)
    {
        cout<<ptr[iCnt]<<endl;
    }


    delete []ptr;

    return 0;
}